app/api/models.py

```diff
@@ -0,0 +1,82 @@
+from enum import Enum
+
+from sqlalchemy import ForeignKey, Integer, String
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from app.core.constance import CATEGORY_NAME_LENGTH, STRING_LENGTH
+from app.core.db import Base
+
+
+class SpeakingSkillEnum(str, Enum):
+    begginer = 'begginer'
+    pre_intermidiate = 'pre-intermidiate'
+    intermidiate = 'intermidiate'
+    pre_advanced = 'pre-advanced'
+    advanced = 'advanced'
+    native = 'native'
+
+
+class TagsEnum(str, Enum):
+    pronunciation = 'pronunciation'
+    speaking = 'speaking'
+    comprehension = 'comprehension'
+    listenig = 'listenig'
+    writing = 'writing'
+    reading = 'reading'
+
+
+class DurationEnum(str, Enum):
+    thirry_minutes = 30
+    fourty_five_minutes = 45
+    sixty_minutes = 60
+    ninty_minutes = 90
+
+
+class Languages(Base):
+    """Model for all languages available."""
+
+    __tablename__ = 'languages'
+
+    name: Mapped[str] = mapped_column(String(STRING_LENGTH), unique=True)
+
+
+class SpokenLanguage(Base):
+    """Model for languages that users can speak."""
+
+    __tablename__ = 'spoken_languages'
+
+    name: Mapped['Languages'] = mapped_column(ForeignKey('language'))
```

Comment:
In ProCharity models they used "categories.id", "users.id" and etc. on `ForeignKey` fields. Dunno why, I guess it just needs specifics